uvplot/io.py

```diff
@@ -38,6 +38,9 @@ def export_uvtable(uvtable_filename, tb, vis="", split_args=None, split=None,
         The CASA::split task must be provided in input as split.
     split : optional
         CASA split task
+    channel : str, optional
+        If 'all', all channels are exported; if 'zero' only the first channel of each spectral window (spw) is exported.
```

Comment:
Instead of `'zero'` I would call it `'first'` which is a clearer statement for taking the first channel.

Reply:
Please propagate the change everywhere, I am not highlighting all occurrences.